tests/test_workflows.py

```diff
@@ -321,3 +323,24 @@ def test_return_exception(app, create_builder):
         "Traceback (most recent call last)"
     )
     assert "ZeroDivisionError: division by zero" in tasks["TASK_ERROR"]["traceback"]
+
+
+@pytest.mark.skip_no_worker()
```

Comment:
I think you can remove this marker, we don't rely on Celery worker for this test.